addon/styles/_pix-input.scss

```
@@ -0,0 +1,91 @@
+.pix-input {
+  @import 'reset-css';
+
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  &__label {
+    font-family: $font-roboto;
+    font-size: 0.875rem;
+    color: $grey-70;
+  }
+
+  &__information {
+    font-family: $font-roboto;
+    font-size: 0.75rem;
+    color: $blue-zodiac;
+    margin-top: 4px;
+  }
+
+  svg.pix-input__icon {
+    position: absolute;
+    bottom: 10px;
+    right: 6px;
+    color: $grey-25;
+    margin-right: 6px;
+  }
+
+  svg ~ svg.pix-input__icon {
+    right: calc(6px + 12px + 2px);
+  }
+
+  svg.pix-input__error-icon {
+    position: absolute;
+    bottom: 10px;
+    right: 6px;
+    color: $white;
+    background: $red;
+    border-radius: 50%;
+    font-size: 0.6rem;
+    padding: 2px;
+    width: 12px;
+    height: 12px;
+  }
+
+  &__error-message {
+    position: absolute;
+    bottom: calc(-4px - 1.5px - 0.75rem);
+    font-family: $font-roboto;
+    font-size: 0.75rem;
+    color: $red;
+    margin-top: 4px;
+  }
+
+  input {
+    margin-top: 4px;
+    padding: 10px 16px;
+    padding-right: 26px;
+    border: 1.5px solid $grey-40;
```

Je pense que c'est une border de `1px` (pour éviter les demi pixel)
```suggestion
    border: 1px solid $grey-40;
```

---

Commentaire lié à https://github.com/1024pix/pix-ui/pull/122#discussion_r692770463 
Du coup je met `1px` en attendant qu'on ait une réponse